assets/scss/templates/_reportes.scss

```diff
@@ -240,8 +269,8 @@
 
 /* Top ranking */
 .wrapper-reports-box {
-  max-width: $dp-wrapper-max-width;
-  margin: $dp-spaces--lv4 auto;
+  /*max-width: $dp-wrapper-max-width;*/
+  /*margin: $dp-spaces--lv4 auto;*/
```

Comment:
What do you think about removing commented code?

Reply:
If I was just generating the version to be able to do tests on the webapp.
Then I delete everything and the word neat